front/pages/api/w/[wId]/invitations/[iId]/index.ts

```diff
@@ -61,8 +61,7 @@ async function handler(
     });
   }
 
-  const invitationId = req.query.iId;
-  let invitation = await getInvitation(auth, { invitationId });
```

Comment:
TS won't be able to infer that `req.query.iId` is of type string even with the previous check on line 54. the object needs to be de-structure and the type assertion run on it.